pkg/groups/service.go

```diff
@@ -382,6 +382,11 @@ func (s *Service) listPermissionsByType(ctx context.Context, ID, pType, continua
 	permissions := make([]string, 0)
 
 	for _, t := range r.GetTuples() {
+		// if relation doesn't start with can_ it means it's not a permission (see #assignee)
+		if !strings.HasPrefix(t.Key.Relation, "can_") {
```

Comment:
this will be needed for identities and roles also
next "tackle" should extract this into a common utilty/helper

Reply:
a role doesn't have any other relationship for now, a role cannot be `assigned` to be another `assignee` role

for identities, true, but not with the current model